src/lib/bcmp/bcmp_config.cpp

```diff
@@ -557,18 +556,17 @@ BmErr bcmp_config_init(Configuration *user_cfg, Configuration *sys_cfg) {
 
   if (user_cfg && sys_cfg) {
     err = BmOK;
-    _usr_cfg = user_cfg;
-    _sys_cfg = sys_cfg;
+    USR_CFG = user_cfg;
+    SYS_CFG = sys_cfg;
   }
 
-  //TODO please handle this better
-  err = packet_add(&config_get, BcmpConfigGetMessage);
-  err = packet_add(&config_set, BcmpConfigSetMessage);
-  err = packet_add(&config_commit, BcmpConfigCommitMessage);
-  err = packet_add(&config_status_request, BcmpConfigStatusRequestMessage);
-  err = packet_add(&config_status_response, BcmpConfigStatusResponseMessage);
-  err = packet_add(&config_delete_request, BcmpConfigDeleteRequestMessage);
-  err = packet_add(&config_delete_response, BcmpConfigDeleteResponseMessage);
-  err = packet_add(&config_value, BcmpConfigValueMessage);
+  bm_err_check(err, packet_add(&config_get, BcmpConfigGetMessage));
```

Comment:
Nice! 👏 

Please queue up a separate task (not this PR) for documenting `bm_err_check`. People using it (including us) need to know (!) that it will fall through and never call the second argument if the first one isn't BmOK. It must only be used with functions that return a BmErr. Think about any common typos someone might make that would still compile, guard against them if possible, but at least document.

Reply:
Task created here:

https://app.shortcut.com/sofar/story/207754/document-bm-err-check-usage-in-bristlemouth-integration-docs